frontend/src/components/mine/Incidents/MineIncidents.js

```diff
@@ -59,10 +59,18 @@ export class MineIncidents extends Component {
   }
 
   handleAddMineIncident = (values) => {
-    this.props.createMineIncident(this.props.mine.mine_guid, values).then(() => {
-      this.props.closeModal();
-      this.props.fetchMineIncidents(this.props.mine.mine_guid);
-    });
+    // set default fatalities & injuries to 0 if not set by user
+    const { number_of_fatalities = 0, number_of_injuries = 0 } = values;
+    this.props
+      .createMineIncident(this.props.mine.mine_guid, {
+        number_of_fatalities,
+        number_of_injuries,
+        ...values,
```

Comment:
I'm not able to think of a case where this would be an issue, but to be a bit more explicit here and to avoid a potential overwrite of the defaulted values, you could pull out the remaining values when you assign the defaults.

```javascript
const { number_of_fatalities = 0, number_of_injuries = 0, ...otherValues } = values;
   this.props
     .createMineIncident(this.props.mine.mine_guid, {
        number_of_fatalities,
        number_of_injuries,
        ...otherValues,
      })
```

Reply:
updated!